guile/mu.scm

```diff
@@ -38,8 +38,7 @@
     mu:header
     ;; message accessors
     mu:field:bcc
-    mu:field:body-html
-    mu:field:body-txt
+    mu:field:body
     mu:field:cc
```

Comment:
Although the Mu Guile API is deprecated, people may depend on it, so wouldn't want to add backward-incompatibilities if it can be avoided.

So, it's okay to remove body-txt/body-html from the documentation, but keep the functions around; the former as a synonym for `body` and the latter always returning `#f`.

Reply:
Makes sense. I'll send an updated version with that change.
Is it ok if I force-push to this branch?

Reply:
I just force-pushed a version implementing the change.